src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -170,6 +170,9 @@ type MoneyRequestConfirmationListProps = {
 
     /** Whether the expense is confirmed or not */
     isConfirmed?: boolean;
+
+    /** Whether the expense is confirming or not */
```

Comment:
Can we be more specific on this one? What are we confirming?

Reply:
@luacmartins not sure how to make it more specific. will this work?

    /** Whether the expense is in the process of being confirmed */